example/BUILD

```diff
@@ -0,0 +1,30 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+
+go_binary(
+  name = "client",
+  srcs = ["client.go"],
+  deps = [
+    "//internal/v2:s2av2",
```

Comment:
Should there be a dependency on ":echo" target as well?

Reply:
Thanks for catching this! I'm not sure it needs to be here, because the echo package has the libraries to run the server. Related to the "go build" error I'm having, to test this, I added the dependency in my workspace. It didn't change the error I get when I run go build. It also didn't affect the bazel run client / server (continues to work correctly)